tracer/src/Datadog.InstrumentedAssemblyGenerator/InstrumentedAssembly.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections.Generic;
+
+namespace Datadog.InstrumentedAssemblyGenerator
+{
+    /// <summary>
+    /// Represents the results info of modified assembly and methods
```

Comment:
```suggestion
    /// Represents the modified assembly on disk, which contains all the metadata 
    /// changes and IL bytecode instrumentation that we performed at runtime.
    
```

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1